_includes/team_table.html

```diff
@@ -0,0 +1,49 @@
+<table style="font-size:medium;">
+    <tr>
+        <th>Role</th>
+        <th>Sub-role</th>
+        <th>Current team</th>
+        <th>Historical contributors</th>
```

Comment:
Here is where to edit if the table headings should be different.

Reply:
Perhaps add footnotes to the headings to explanations at the bottom of the table (along the lines of what Irfan is suggesting).